docs/en/04_Changelogs/4.10.0.md

```diff
@@ -1,41 +1,170 @@
+| silverstripe/fulltextsearch | 3.9.0 |
+| silverstripe/gridfieldqueuedexport | 2.6.1 |
+| silverstripe/html5 | 2.2.0 |
+| silverstripe/hybridsessions | 2.4.0 |
+| silverstripe/iframe | 2.2.0 |
+| silverstripe/ldap | 1.3.0 |
+| silverstripe/mfa | 4.5.1 |
+| silverstripe/realme | 4.2.0 |
+| silverstripe/registry | 2.4.0 |
+| silverstripe/restfulserver | 2.4.0 |
+| silverstripe/security-extensions | 4.2.1 |
+| silverstripe/securityreport | 2.4.0 |
+| silverstripe/segment-field | 2.4.0 |
+| silverstripe/sharedraftcontent | 2.6.0 |
+| silverstripe/sitewidecontent-report | 3.2.0 |
+| silverstripe/spamprotection | 3.2.0 |
+| silverstripe/spellcheck | 2.3.0 |
+| silverstripe/subsites | 2.5.0 |
+| silverstripe/tagfield | 2.8.0 |
+| silverstripe/taxonomy | 2.3.0 |
+| silverstripe/textextraction | 3.3.0 |
+| silverstripe/totp-authenticator | 4.3.0 |
+| silverstripe/userforms | 5.11.1 |
+| silverstripe/versionfeed | 2.2.0 |
+| silverstripe/webauthn-authenticator | 4.4.1 |
+| silverstripe/widgets | 2.2.0 |
+| symbiote/silverstripe-advancedworkflow | 5.6.0 |
+| symbiote/silverstripe-multivaluefield | 5.2.0 |
+| symbiote/silverstripe-queuedjobs | 4.9.0 |
+| tractorcow/silverstripe-fluent | 4.6.0 |
+
+</details>
+
 
 ## Regression test and Security audit{#audit}
 
 This release has been comprehensively regression tested and passed to a third party for a security-focused audit.
 
 While it is still advised that you perform your own due diligence when upgrading your project, this work is performed to ensure a safe and secure upgrade with each recipe release.
 
-## Dropping support for PHP 7.1 and PHP 7.2{#phpeol}
+## PHPUnit 9.5 and PHP 8.0 official support{#php8}
+
+Previous Silverstripe CMS Recipe releases could be run against PHP 8.0. However, the Silverstripe CMS Recipe 4.10.0 is the first release to officially support PHP 8.0. 
+
+The major road block to official PHP 8.0 support was our reliance on a deprecated version of PHPUnit. Adding support for PHPUnit 9.5 removed this road block. All core module tests have been rewritten to run with PHPUnit 9.5.
+
+### What about my project test suite?
+
+Silverstripe CMS Recipe 4.10.0 retains support for the `sminnee/phpunit` fork of PHPUnit 5.7. So your project's tests can still be run without modification ... as long as you are not using PHP 8.0. If you want your own tests to run with PHP 8.0, then you have have to upgrade to PHPUnit 9.5 and update your tests to be compatible with the latest PHPUnit API.
+
+Our intention is to removed the PHPUnit 5.7 compatibility layer in the first Silverstripe CMS release of 2023. This gives projects about a year to migrate their test coverage to PHPUnit 9.5.
+
+Read [Upgrading to PHPUnit 9.5 for PHP 8 support](/upgrading/phpunit9/) for more details on how to migrate to PHPUnit 9.5.
+
+Jump on the [Silverstripe CMS forum](https://forum.silverstripe.org/tag/phpunit95) if you have questions about converting your tests to be compatible with PHPUnit 9.5.
```

Comment:
I've left an [introductory post on the forum](https://forum.silverstripe.org/t/phpunit-9-5-migration-when-upgrading-to-silverstripe-cms-recipe-4-10/4781) under that tag to encourage people to share their questions and tips. I've also left a bit more informal suggestion to help people get started.